src/content/en/updates/2020/01/devtools.md

```diff
@@ -0,0 +1,153 @@
+  **Show Device Frame**).
+* [Throttle the network and CPU](http://localhost:8080/web/tools/chrome-devtools/device-mode/#throttle)
+  to more accurately simulate a mobile user's web browsing conditions.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/49e4673403214c99cc9d358fb5f311727dcf47e8 #}
+
+[Chromium issue #924693](https://crbug.com/924693)
+
+## Cookie-related updates {: #cookies }
+
+### Blocked cookies in the Cookies pane {: #blockedcookies }
+
+The Cookies pane in the Application panel now colors blocked cookies with a yellow background.
+
+![Blocked cookies in the Cookies pane of the Application panel](/web/updates/images/2020/01/blockedcookies.png)
+
+See also [Debug why a cookie was blocked](/web/updates/2019/10/devtools#blockedcookies) to learn
+how to access a similar UI from the Network panel.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/341eea56a810370d6f36ef6dd750df9cd66cd398 #}
+
+[Chromium issue #1030258](https://crbug.com/1030258)
+
+### Cookie priority in the Cookie pane {: #cookiepriority }
+
+The Cookies tables in the Network and Application panels now include a **Priority** column.
+
+<aside class="caution">
+  Chrome is the only browser that supports cookie priority.
+</aside>
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/e65507201ae7c4b83f297bab97e4f4dcc1e1168d #}
+
+[Chromium issue #1026879](https://crbug.com/1026879)
+
+### Edit all cookie values {: #edit }
+
+All cells in the Cookie tables are editable now, except cells in the **Size** column because that
+column represents the network size of the cookie, in bytes. 
+See [Fields](/web/tools/chrome-devtools/storage/cookies#fields) for an explanation of each column.
+
+![Editing a cookie value](/web/updates/images/2020/01/editcookie.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/cc93308cee322f28b02be92ac410d6651938de37 #}
+
+### Copy as Node.js fetch to include cookie data {: #fetchcookies }
+
+Right-click a network request and select **Copy** > **Copy as Node.js fetch** to get a
+`fetch` expression that includes cookie data.
+
+![Copy as Node.js fetch](/web/updates/images/2020/01/fetchcookies.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/7c50d1f484d4fd6d308b7e5b825fc21c73830a92 #}
+
+[Chromium issue #1029826](https://crbug.com/1029826)
+
+## More accurate manifest icons {: #manifesticons }
+
+Previously, the Manifest pane in the Application panel would perform its own requests in order to
+display web app manifest icons. DevTools now shows the exact same manifest icon that Chrome uses.
+
+![Icons in the Manifest pane](/web/updates/images/2020/01/manifesticons.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/3c387a84cd3012165f279b95608f17b3c826ed4b #}
+
+[Chromium issue #985402](https://crbug.com/985402)
+
+## Hover over content properties to see unescaped values {: #content }
```

Comment:
@mathiasbynens not sure if this feature applies to properties other than `content`

Reply:
Only `content` for now, since that was the most pressing issue for the Material/Fonts team. We can expand support to other CSS properties that accept string values over time.

Reply:
Cool, content is fine as is for now in that case